src/__experimental__/components/checkable-input/checkable-input.style.js

```diff
@@ -40,6 +45,19 @@ const StyledCheckableInputWrapper = styled.div`
       }
     }
 
+    ${StyledCheckableInput},
+    ${HiddenCheckableInputStyle},
+    ${StyledCheckableInputSvgWrapper},
+    svg {
+      display: flex;
+      flex: 0 1 16px;
```

Comment:
default flex is:
`flex: 0 1 auto`
I think we can replace it with `flex-basis: 16px`;

Reply:
resolved in f37ec57f3d23c58afbcb46ab4bd5cf5672be04c0